apps/etterlatte-api/src/test/kotlin/no/nav/etterlatte/samordning/vedtak/SamordningVedtakServiceTest.kt

```diff
@@ -394,7 +394,7 @@ fun vedtak(
         status = VedtakStatus.ATTESTERT,
         virkningstidspunkt = virkningstidspunkt,
         sak = VedtakSak(ident = "123", sakstype, id = randomSakId()),
-        behandling = Behandling(id = UUID.randomUUID(), type = BehandlingType.FØRSTEGANGSBEHANDLING),
+        forenkletRevurderingInfo = ForenkletRevurderingInfo(id = UUID.randomUUID(), type = BehandlingType.FØRSTEGANGSBEHANDLING),
```

Comment:
Navnet her skurrer litt for meg, `ForenkletRevurderingInfo` inneholder referanse til en førstegangsbehandling? Da synes jeg den forrige står seg bedre?

Reply:
Ja, bare forslag. Er litt usikker på den, for det er jo kanskje revurdering og hvis det er det så får du med den dataen. Trenger vi det egentlig?